Dockerfile

```diff
@@ -1,29 +1,39 @@
-FROM ubuntu:bionic
+FROM php:7.3-apache
 
-# TODO: think through optimizations here once this is working.
-RUN    apt-get update \
-    && apt --yes install software-properties-common \
-    && apt-add-repository --yes --update ppa:ansible/ansible \
-    && apt --yes install git ansible
+# Extensions we know LF requires/desires:
+# curl - already provided in image
+# mbstring - already provided in image
+# xdebug
+# intl
+# gd
+# mongodb
+#
+# see https://github.com/mlocati/docker-php-extension-installer
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+RUN install-php-extensions gd xdebug mongodb curl intl
+RUN install-php-extensions @composer
 
-# make a working area, an "app" area
+# install Language Forge APT dependencies
+RUN apt-get update && apt-get -y install p7zip-full unzip
+
+# Pull in Language Forge git repo
 RUN mkdir -p /data
 WORKDIR /data
-
-# go ahead and pull local filesystem in so we can take care of more manual configs
-# TODO: need to optimize this by only copying things that are needed for the set up
 COPY . /data
 
-# not sure whether to handle this outside of this image, i.e., tell user to git clone ... --recurse-submodules, or just do it here to make sure the submodules are guaranteed to be here for ansible scripts...
-RUN git submodule update --init --recursive
 
-# time to run the ansible scripts
-WORKDIR /data/deploy
 
-# TODO: need to make a distinction between environment setup commands and those commands meant for running the services.
-# TODO: left off the -K otherwise this command becomes interactive (password requested)
-# RUN ansible-playbook playbook_bionic.yml --limit localhost
+# apt packages to investigate further
+#   - default-jre-headless
+#   - lfmerge
```

Comment:
this one we still has my curiosity piqued...we may be able to run it in a separate container and use some form of `rsh/rexec` from the php container to the lfmerge container for issuing those commands against another "volume-shared" file system.  Docker sets up a private network between the containers so this should be possible as far as I can think without trying it.

Ideally we'd have a daemon-like process listening for some protocol to issue the commands appropriately.